### ISSUE TITLE
Null pointer exception if you have a null value in a condition

### ISSUE THREAD

**Issue by bi0qaw** [MundoSK/MundoSK#18]
MundoSk throws a null pointer exception if you try to evaluate a condition with a null value.

### Bug Report

Skript Version: v2.2-dev32d

MundoSK Version: v1.8.3

Minecraft Version: 1.12

### Steps to Reproduce the Issue
```
command /test:
    trigger:
        if {_var}:
            send "message"
```

### Console Errors (if applicable)
```
[15:59:14 ERROR]: #!#! 
[15:59:14 ERROR]: #!#! Stack trace:
[15:59:14 ERROR]: #!#! java.lang.NullPointerException
[15:59:14 ERROR]: #!#!     at com.pie.tlatoani.Miscellaneous.CondBoolean.check(CondBoolean.java:18)
[15:59:14 ERROR]: #!#!     at ch.njol.skript.lang.Condition.run(Condition.java:56)
[15:59:14 ERROR]: #!#!     at ch.njol.skript.lang.Conditional.walk(Conditional.java:51)
[15:59:14 ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89)
[15:59:14 ERROR]: #!#!     at ch.njol.skript.lang.Trigger.execute(Trigger.java:55)
[15:59:14 ERROR]: #!#!     at ch.njol.skript.command.ScriptCommand.execute2(ScriptCommand.java:266)
[15:59:14 ERROR]: #!#!     at ch.njol.skript.command.ScriptCommand.execute(ScriptCommand.java:230)
[15:59:14 ERROR]: #!#!     at ch.njol.skript.command.ScriptCommand.onCommand(ScriptCommand.java:181)
[15:59:14 ERROR]: #!#!     at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44)
[15:59:14 ERROR]: #!#!     at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141)
[15:59:14 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(CraftServer.java:641)
[15:59:14 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchServerCommand(CraftServer.java:627)
[15:59:14 ERROR]: #!#!     at net.minecraft.server.v1_12_R1.DedicatedServer.aP(DedicatedServer.java:443)
[15:59:14 ERROR]: #!#!     at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406)
[15:59:14 ERROR]: #!#!     at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:678)
[15:59:14 ERROR]: #!#!     at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[15:59:14 ERROR]: #!#!     at java.lang.Thread.run(Thread.java:745)
[15:59:14 ERROR]: #!#! 
[15:59:14 ERROR]: #!#! Version Information:
[15:59:14 ERROR]: #!#!   Skript: 2.2-dev32d
[15:59:14 ERROR]: #!#!   Bukkit: 1.12-R0.1-SNAPSHOT
[15:59:14 ERROR]: #!#!   Minecraft: 1.12
[15:59:14 ERROR]: #!#!   Java: 1.8.0_102 (Java HotSpot(TM) 64-Bit Server VM 25.102-b14)
[15:59:14 ERROR]: #!#!   OS: Mac OS X x86_64 10.11.6
[15:59:14 ERROR]: #!#! 
[15:59:14 ERROR]: #!#! Running CraftBukkit: false
[15:59:14 ERROR]: #!#! Running Spigot (or compatible): true
[15:59:14 ERROR]: #!#! Running Paper (or compatible): false
[15:59:14 ERROR]: #!#! 
[15:59:14 ERROR]: #!#! Current node: null
[15:59:14 ERROR]: #!#! Current item: {_var}
[15:59:14 ERROR]: #!#! Thread: Server thread
[15:59:14 ERROR]: #!#! 
[15:59:14 ERROR]: #!#! Language: english
[15:59:14 ERROR]: #!#! Link parse mode: DISABLED
[15:59:14 ERROR]: #!#! 
[15:59:14 ERROR]: #!#! End of Error.
[15:59:14 ERROR]: #!#! 
```

**Comment by Syst3ms** [comment]
I legit just had to think about this.
`Boolean` can be null, so auto-unboxing to `boolean` fails with `null`.
`if (b != null && b)` will work.

**Comment by TlatoaniHJ** [comment]
Fixed in MundoSK 1.8.5.
